### PR TITLE
[Merged by Bors] - 0.10: AccessKit integration into bevy_ui

### PR DESCRIPTION
**Pull request by alice-i-cecile**
🎉 

I've deliberately left out a full code example for now. This is a) almost certain to change by the time 0.11 comes b) is quite clunky c) requires some [trickery](https://github.com/bevyengine/bevy/pull/7894) to get alt text displaying properly.

I care a lot about making this better, so stay tuned.

### CODE REVIEW

**Comment by alice-i-cecile** [comment]
FYI @ndarilek @mwcampbell :) Thanks again. If you have time, feel free to suggest changes and I'll merge them in!

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 1]
```diff
@@ -1291,6 +1291,24 @@ fn run_or_attack(
 [`AnimationPlayer`]: https://docs.rs/bevy/0.10.0/bevy/animation/struct.AnimationPlayer.html
 [`play_with_transition`]: https://docs.rs/bevy/0.10/bevy/animation/struct.AnimationPlayer.html#method.play_with_transition
 
+### AcessKit integration into `bevy_ui`
+
+<div class="release-feature-authors">authors: @ndarilek</div>
+
+Games are for everyone: and the way they're built should reflect that.
+Acessible games are rare, and proper support is often an afterthought, both at an engine and a game level.
```
Comment (cart) [inline content/news/2023-03-04-bevy-0.10/index.md:1299]:
```suggestion
Accessible games are rare, and proper support is often an afterthought, both at an engine and a game level.
```

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 2]
```diff
@@ -1291,6 +1291,24 @@ fn run_or_attack(
...
+By building our UI solution with acessibility in mind, we hope to fix that.
```
Comment (ndarilek) [inline content/news/2023-03-04-bevy-0.10/index.md:1300]: s/acessibility/accessibility/ Sorry, not sure how to make that a suggested change. Thanks for writing this up!

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 3]
```diff
@@ -1291,6 +1291,24 @@ fn run_or_attack(
...
+
+Bevy has [joined `egui`](https://github.com/emilk/egui/pull/2294) in making the first steps towards cross-platform accessibility-by-default, with the help of the outstanding [AccessKit](https://github.com/AccessKit/accesskit) crate.
+To our knowledge, this makes Bevy the first general purpose game engine with first-party acessibility support.
```
Comment (ndarilek) [inline content/news/2023-03-04-bevy-0.10/index.md:1303]: s/acessibility/accessibility/ here too.

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 4]
```diff
@@ -1291,6 +1291,24 @@ fn run_or_attack(
...
+
+We've exposed Bevy's UI hierarchy and text elements to screen readers and other assistive devices, managed by the new on-by-default `bevy_a11y` crate.
+This is ultimately powered by the new [`AccesibilityNode`](https://dev-docs.bevyengine.org/bevy/a11y/struct.AccessibilityNode.html) component, which combines with the existing hierarchy to expose this information directly to AccessKit and the [`Focus`](https://dev-docs.bevyengine.org/bevy/a11y/struct.Focus.html) resource, which stores the entity that has focus.
```
Comment (ndarilek) [inline content/news/2023-03-04-bevy-0.10/index.md:1306]: s/accesibility/accessibility/. Also might be worth clarifying that this is specifically keyboard focus.

**Review by ndarilek — COMMENTED** [review]
Thanks again for writing this up!

File: content/news/2023-03-04-bevy-0.10/index.md (hunk 4 above)
Comment (alice-i-cecile) [reply content/news/2023-03-04-bevy-0.10/index.md:1306]:
```suggestion
This is ultimately powered by the new [`AccessibilityNode`](https://dev-docs.bevyengine.org/bevy/a11y/struct.AccessibilityNode.html) component, which combines with the existing hierarchy to expose this information directly to AccessKit and the [`Focus`](https://dev-docs.bevyengine.org/bevy/a11y/struct.Focus.html) resource, which stores the entity that has keyboard focus.
```

File: content/news/2023-03-04-bevy-0.10/index.md (hunk 3 above)
Comment (alice-i-cecile) [reply content/news/2023-03-04-bevy-0.10/index.md:1303]:
```suggestion
To our knowledge, this makes Bevy the first general purpose game engine with first-party accessibility support.
```

**Review by cart — APPROVED** [review]

**Pull request closed without merging**